.github/workflows/nightly.yml

```diff
@@ -121,6 +121,8 @@ jobs:
         run: |
           cp ${{ github.workspace }}/.github/workflows/nightly_notes.md "${RUNNER_TEMP}/nightly_notes.md"
           cd nightly_release
+          echo '' >> "${RUNNER_TEMP}/nightly_notes.md"
```

Comment:
Why is this needed?

Reply:
To have a an empty space between header and text